devices.js

```diff
@@ -8480,6 +8480,40 @@ const devices = [
             }
         },
     },
+    {
+        zigbeeModel: ['URC4450BC0-X-R'],
+        model: 'URC4450BC0-X-R',
+        vendor: 'Xfinity',
+        description: 'Alarm Security Keypad',
+        supports: 'action, arm',
+        meta: {configureKey: 1, battery: {voltageToPercentage: '3V_2100'}},
+        fromZigbee: [fz.command_arm, fz.temperature, fz.battery, fz.ias_occupancy_alarm_1, fz.identify, fz.ias_contact_alarm_1,
+            fz.ias_occupancy_alarm_2, fz.ias_occupancy_alarm_1_with_timeout],
+        exposes: [e.battery(), e.voltage(), e.occupancy(), e.battery_low(), e.tamper(), e.presence(),
+            exposes.numeric('action_code', 1), exposes.text('action_zone', 1), e.temperature(), e.action([
```

Comment:
In from Zigbee I see `fz.ias_contact_alarm_1` but there is no `e.contact()`.

Reply:
Updated to include e.contact()